content/en/docs/troubleshooting.md

```diff
@@ -1,6 +1,88 @@
 ---
 title: "Troubleshooting"
-description: "Troubleshooting common problems with Notary"
+description: "Troubleshooting common problems with Notation"
 type: docs
 weight: 7
----
\ No newline at end of file
+---
+
+> NOTE:
+>
+> Suggest a troubleshooting item by raising a PR to our [documentation repository](https://github.com/notaryproject/notaryproject.dev)
+
+## Enabling notation CLI commands logging
+
+Printing log messages for debugging the problems can be enabled using the `--debug` or `-d` flag. Using the `--verbose` or `-v` flag to enable verbose logging will be enough for most cases.
+
+For example:
+
+```shell
+# Enable the debugging logs
```

Comment:
What is the difference between debug and verbose? It does seem redundant although we print different information for each one. Clarifying when to use one or the other would be helpful.